internal/pkg/deploy/cloudformation/stack/pipeline.go

```diff
@@ -4,10 +4,10 @@
 package stack
 
 import (
-	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/template"
 	"github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/aws/copilot-cli/internal/pkg/template"
 
-	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/deploy"
+	"github.com/aws/copilot-cli/internal/pkg/deploy"
```

Comment:
The deploy should come before template?

Reply:
I'm not sure but go fmt seems to be okay with it!